ansible/run.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: local
+  connection: local
+  vars_files:
+      - runtime_vars.yml
+  roles:
+      - facts
+      - nodejs
```

Comment:
@avtar, I agree this would be desirable to have (maybe configurable) - I'm having issues getting it to happen in practice with the role not respecting the variables I pass it. Uncertain why but will continue debugging.

I'd be interested in thoughts about what should happen long-term with container testing - is it maybe appropriate that "test a launched container" be a role for a sidecar container, similar to what we'd like to do longer term with loading the initial data?